src/progress_bar.jl

```diff
@@ -36,17 +34,16 @@ function next!(p::ProgressBar, io::IO=stdout)
     elapsed_time_str = string(elapsed_time ÷ 3600, "h ", lpad((elapsed_time % 3600) ÷ 60, 2, "0"), "m ", lpad(elapsed_time % 60, 2, "0"), "s")
 
     # Calculate the estimated time of arrival
-    eta = floor(Int, elapsed_time ÷ counter * (max_counts - counter))
+    eta = floor(Int, elapsed_time / counter * (max_counts - counter))
```

Comment:
just a small comment, would it be better to change this line as
```julia
eta = elapsed_time * (max_counts - counter) ÷ counter
```
since `elapsed_time`, `max_counts`, `counter` are all `integers`
by using floor division (`÷`), we don't need to do `floor()` again after normal division (`/`) ?

Reply:
The rpevious case had some issues, because it showed ETA = 0s while it was supposed to be 40s ish. So I thought to fix this problem in this way